requirements-dev.txt

```diff
@@ -5,4 +5,5 @@ pdbpp
 pyflakes
 pytest
 pytest-cov
-sphinx
+sphinx>=1.6.0
+niftools_sphinx_theme
```

Comment:
Probably good to stick a version here also.
Where is this sourced from seeing as it has "niftools", its not one of our projects.

Reply:
It's a sphinx theme I created from copying your niftools.github.io repository so that all the documentation would look the same as the main website.